### ISSUE TITLE
How to find jobs outside London

### ISSUE THREAD

**Issue by HussienAlasker** [HussienAlasker/JobLang-#1]
Are you looking for jobs in United Kingdom? Usually the largest concentrations of jobs are in in the capital which is London (You can view all London jobs just on joblang).In addition, some people prefer to go and work in other small cities where it  is cheaper  to rent a house and everything is within walking distance. You can check our jobs in the following locations.
Regard the place that you want to work in. There are two main points we need to highlight. Some of the companies do not post their jobs and they try to find people online. Therefore, we need to have full view about the terms and keywords employers are searching for to find people and candidates that have the required skills and qualifications in order to perform the job perfectly. This task or research requires using tools that can give you statistics about what people are search for in specific area. These tools are called Keyword tools such as Google Keyword research tools.
After figuring what keywords are being used by employers who are hunting the candidates? You need to optimize your CV and profile with these terms to het high visibility of your Cv.
So, how Google and other search engines identify which profiles should appear on the top of search results? The two factors are relevancy and website rank. Relevancy is when your CV have the terms that user search for and Rank it vary depend on the terms and keywords, some websites are more authoritative than other websites for specific terms
Another benefit of Keyword research, you can perform many researches on different job boards and search engines to get the highest number of jobs that match your profile.
In conclusion, you need to optimize your CV with Keywords and perform searches on job boards and search engines using these terms.

https://joblang.com/job/en
